packages/wdio-local-runner/src/run.ts

```diff
@@ -7,30 +7,38 @@ import { SHUTDOWN_TIMEOUT } from './constants'
 
 const log = logger('@wdio/local-runner')
 
-export const runner = new Runner()
+/**
+ * ToDo(Christian): remove when @wdio/runner got typed
+ */
+interface RunnerInterface extends NodeJS.EventEmitter {
+    sigintWasCalled: boolean
+    [key: string]: any
```

Comment:
This line can be removed as it already extends the object, so there's no need to tell TS that there will be more properties

Reply:
a runner instance (which is a child of eventemitter) can have arbitrary commands which is why we actually need this, particularly in the line where we call that arbitrary command:

```ts
runner[m.command as string](m).then(
```